src/ripple/rpc/handlers/GetCounts.cpp

```diff
@@ -120,11 +123,7 @@ getCountsJson(Application& app, int minObjectCount)
     textTime(uptime, s, "second", 1s);
     ret[jss::uptime] = uptime;
 
-    ret[jss::node_writes] = app.getNodeStore().getStoreCount();
-    ret[jss::node_reads_total] = app.getNodeStore().getFetchTotalCount();
-    ret[jss::node_reads_hit] = app.getNodeStore().getFetchHitCount();
-    ret[jss::node_written_bytes] = app.getNodeStore().getStoreSize();
-    ret[jss::node_read_bytes] = app.getNodeStore().getFetchSize();
+    ret[jss::nodestore] = app.getNodeStore().getCountsJson();
```

Comment:
This changes the output formatting of the get_counts RPC call to have a sub-object called "nodestore" that contains nodestore counters. The function Database::getCountsJson() was created to allow both get_counts and server_info counters to access this data. The following commit reverts get_counts so that the counters are no longer in a new object.
https://github.com/mtrippled/rippled/commit/3faaa437b47726d6b1141070238dd63448a29d6a